src/index.ts

```diff
@@ -57,7 +57,13 @@ async function main(): Promise<void> {
 		// Do this *after* to ensure messages sent during installation aren't dropped.
 		await installAll();
 	} else {
-		await installAll();
+		if (!onlyTestTsNext) {
+			await installAll();
+		} else {
+			// if `onlyTestTsNext` is true we only need to install the next typescript version
```

Comment:
Remove this comment. The names onlyTestTsNext and installNext convey the intent by themselves.